test/unit/auth_sources/auth_source_ldap_test.rb

```diff
@@ -6,33 +6,12 @@ def setup
     User.current = users(:admin)
   end
 
-  test "should exists a name" do
```

Comment:
Appears to have been fixed at https://github.com/theforeman/foreman/pull/2651/files#diff-6c1e29f55a935fec94f5837557af0c7aR10